json/testnet-bitcoin-config.json

```diff
@@ -1,13 +1,13 @@
 {
   "bitcoinFeeSpendingCutoffPeriodInBlocks": 1,
   "bitcoinFeeSpendingCutoff": 0.001,
-  "bitcoinPeerUri": "http://localhost:18332",
+  "bitcoinPeerUri": "http://bitcoin-node:18332",
   "bitcoinRpcUsername": "",
   "bitcoinRpcPassword": "",
-  "bitcoinWalletOrImportString": "[FILL THIS IN!]",
+  "bitcoinWalletOrImportString": "cUuEua7N4cfCfjceHreEJ5k6TNtV9tko4q7hDwJ1t4qfvoAdjVQY",
```

Comment:
Should leave as it was? I hope this is not a key of importance? Maybe it's implied.

Reply:
If this isn't set then the config that is baked into it for running the cluster of services in testnet wont work, we either have to decide between

No configs in the image at all and hence force the call to supply their own when initialising the services
Have a default config for testnet, like this approach is showing with a wallet that is auto refuelled by a faucet.

IMO option 2 does reduce friction for people experimenting.

Reply:
I am bought in to the idea of having auto refuelled faucet, can you teach me how that's done? Is there some public available site that can do this for us?